tmt/steps/prepare/install.py

```diff
@@ -134,6 +153,52 @@ def enable_copr(self, command, plugin, guest):
                 self.info('copr', copr, 'green')
                 guest.execute(f'{command} copr enable -y {copr}')
 
+    def enable_custom_repo(self, command, guest):
+        """ Enable requested repositories """
+        repos = self.get('repo'):
+        if not repos:
+            return
+
+        # Create repo file content one bye one
+        repo_list = list()
+        for repo_id, repo in enumerate(repos):
+            self.debug(f'Create repo file content for {repo}.')
+            gpgcheck = 1 if self.get('gpgcheck', '0') == '1' else 0
+            skip_if_unavl = 1 if self.get('skip-if-unavailable', '0') == '1'
+                            else 0
```

Comment:
I think these comparisons may bite you. After being loaded from fmf file, which is basically a YAML file, are you sure that the value in `gpgcheck: 1` is really a string? It may very well be an integer, failing the `== '1'` test.

Personally, I'd prefer booleans here, `true`, `false`. `gpgcheck: false` is more human-friendly than `gpgcheck: 0` which is pretty much yum/dnf forcing others to follow its integers-for-booleans view of the universe... We can make things nice for humans, let's not punish them for what yum does :)

Using booleans would help with tests as well, `gpgcheck = self.get('gpgcheck', False)` and you're done.

Reply:
Good point. True/false is easier to document as well.

Reply:
@happz and @lukaszachy, Ansible [yum_repository_module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_repository_module.html) uses `yes/no` to set `gpgcheck` and `skip_if_unavailable`, do we follow it OR just use `true/false`?

Reply:
If I'm not mistaken, in theory, plenty of "positive" and "negative" tokens are supported by YAML itself, see https://yaml.org/type/bool.html - e.g. `yes`, `true` and `on` should manifest as Python's `True` when loaded by YAML parser. But, I'd stick to what TMT docs use for various boolean options, and `true`/`false` seem to be preferred.

Good thing is, you don't have to worry about it, except for documentation where `true` and `false` seem to be the preferred choice. In code itself, you can expect `self.get('gpgcheck', False)` to deliver a valid `bool` instance, `True` or `False`, no matter what was written in fmf input, as long as it was a valid boolean representation. You should get `True` for true-ish inputs and vice versa.

I'd therefore use `true` and `false` in docstrings and examples, but wouldn't check for any particular boolean representation understood by YAML in my code, because if `yes` is accepted by the YAML parser, I would not get `yes` on my table anyway, `get()` would return `True` for all those `yes`, `true` and `ON` inputs.